### ISSUE TITLE
Downloaded replays is impossible to manage

### ISSUE THREAD

**Issue by Allovinci** [jeffhill/Dota2#1428]
### Description

TL;DR:
The whole replay manage system is broken: it's nearly impossible to find a downloaded replay.

Specifically:
(1) The "Saved to watch later" doesn't show any recently downloaded replays. Have to click in "Downloads" to view just downloaded rep
![image](https://user-images.githubusercontent.com/107814348/174499760-e58048c0-87e6-4778-ac08-953eed4116a1.png)

(2) In "Downloads", No useful match information were shown including match length, match details (drafts, which side win, which tournament is this rep from and which team is playing, etc.).
![image](https://user-images.githubusercontent.com/107814348/174499889-e140a319-9a98-4eb1-8e68-02c81f1b3157.png)

### Screenshots

![image](https://user-images.githubusercontent.com/107814348/174499860-1466cbd3-0e20-4735-938e-6d4a2b1c219c.png)
![image](https://user-images.githubusercontent.com/107814348/174499889-e140a319-9a98-4eb1-8e68-02c81f1b3157.png)

**Comment by Aluerie** [comment]
Pretty much duplicate #349 with quite a bit of upvotes

So please, close your issue, upvote + comment those extra details in linked issue

**Comment by Allovinci** [comment]
Searched for "rep" and "replay" didn't find that issue. Will close this and upvote #349. Thanks.
